src/components/ContributionsEpic.tsx

```diff
@@ -227,6 +229,30 @@ export const ContributionsEpic: React.FC<Props> = ({
                     className={paymentImageStyles}
                 />
             </div>
+
+            {showReminderFields && (
+                <ContributionsEpicReminder
+                    reminderDate={showReminderFields.reminderDate}
+                    reminderDateAsString={showReminderFields.reminderDateAsString}
+                />
+            )}
         </section>
     );
 };
+
+export default {
+    name: 'Components/ContributionsEpic',
+    Component: ContributionsEpic,
+    getInitScript: (props: Props): string | undefined => {
+        if (props.variant.showReminderFields) {
```

Comment:
We want this function to be smart enough to only return the JS that's needed (if any), so we call it with the component's props.